src/electionguard/serializable.py

```diff
@@ -17,6 +17,7 @@
     default_nonetype_deserializer,
 )
 
+S = TypeVar("S", bound="Serializable")
```

Comment:
This is a trick for supporting type hints for factory classmethods:
https://github.com/python/typing/issues/58#issuecomment-326240794